docs/manual/determinism-checker.tex

```diff
@@ -432,6 +433,33 @@
 E set(@PolyDet List<E> this, @PolyDet("use") int index, E element)
 \end{Verbatim}
 
+As alluded to earlier, we allow \<@PolyDet("use")> to resolve to any
```

Comment:
It's not _any_ qualifier. `@PolyDet("use")` is resolved as a separate polymorphic qualifier. If that qualifier is a supertype of the instantiation of `@PolyDet`, then the instantiation of `@PolyDet` is used as the instantiation of `@PolyDet("use")`.